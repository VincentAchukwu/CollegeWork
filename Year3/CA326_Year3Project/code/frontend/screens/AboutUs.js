import React from "react";
import { StyleSheet, Text, View, StatusBar, ScrollView} from "react-native";
import Hyperlink from 'react-native-hyperlink';

const AboutUs = () => {
    return (
        <View style={styles.container}>
            <StatusBar backgroundColor="#1E90FF" barStyle="light-content"/>

            <ScrollView>
                <View style= {styles.header}>
                    <Text style= {styles.headerContent}> Who We Are ? </Text>
                </View>
                <View style= {styles.body}>
                    <Text style= {styles.bodyContent}>
                        Hello, Our names are Tinh Lien and Vincent Achukwu and we're currently
                        in our 3rd year of Computer Science in Dublin City University. This Currency
                        Conversion App is made as our 3rd year project. The goal of this application
                        is to give you the updated rates on currencies, along with the ability to view the
                        previous rates of currencies.
                    </Text>
                </View>

                <View style= {styles.header}>
                    <Text style= {styles.headerContent}> Links to Our Socials </Text>
                </View>
                <View style= {[styles.body, {marginTop: "3%"}]}>
                    <Text style= {[styles.bodyContent, {fontWeight: "bold"}]}>
                        Tinh Lien:
                    </Text>
                    <Hyperlink linkDefault={ true }>
                        <Text style= {{color: "blue", marginTop: "1%"}}> https://github.com/TinhLien </Text>
                        <Text style= {{color: "blue"}}> https://www.linkedin.com/in/tinh-lien-2bb7581b6/ </Text>
                    </Hyperlink>
                </View>
                <View style= {[styles.body, {marginTop: "4%"}]}>
                    <Text style= {[styles.bodyContent, {fontWeight: "bold"}]}>
                        Vincent Achukwu:
                    </Text>
                    <Hyperlink linkDefault={ true }>
                        <Text style= {{color: "blue", marginTop: "1%"}}> https://github.com/VincentAchukwu </Text>
                        <Text style= {{color: "blue"}}> https://www.linkedin.com/in/vincent-achukwu/ </Text>
                    </Hyperlink>
                </View>

            </ScrollView>
        
        </View>
        )
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        backgroundColor: "#ffffff"
    },
    header: {
        marginTop: "6%",
        justifyContent: "center",
        alignItems: "center",
    },
    headerContent: {
        fontSize: 24,
        fontWeight: "bold",
        color: "#1E90FF",
    },
    images: {
        marginTop: "4%"
    },  
    body: {
        justifyContent: "center",
        alignItems: "center",
        paddingLeft:"8%",
        paddingRight: "8%"
    },
    bodyContent: {
        fontSize: 16,
    }
})

export default AboutUs;