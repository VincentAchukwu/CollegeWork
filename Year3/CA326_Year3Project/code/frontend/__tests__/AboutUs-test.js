import "react-native";
import React from "react";
import AboutUs from "../screens/AboutUs";
import renderer from "react-test-renderer";

test("AboutUs Snapshot", () =>{
    const snap = renderer.create(
        <AboutUs/>
    ).toJSON;

expect(snap).toMatchSnapshot();
})