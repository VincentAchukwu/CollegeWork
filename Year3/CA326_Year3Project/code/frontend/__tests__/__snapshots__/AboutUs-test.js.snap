// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs Snapshot 1`] = `[Function]`;
